apps/send_event/src/main.rs

```diff
@@ -153,7 +150,7 @@ mod tests {
 
     #[test]
     fn can_send_and_receive_event() {
-        let expected = c"Hello";
+        let expected = c"Hello\xc3\x28";
```

Comment:
Consider also trying to create a `&str` from this input to illustrate that we can not assume valid utf-8 in strings.

Or adding a comment to the same effect in case the information gets lost to time.

Reply:
Good point, will fix on Monday.